src/utils/generate-report.ts

```diff
@@ -18,12 +18,12 @@ export interface ValidationResult {
     }>;
 }
 
-export async function generateReport(
-    outFile: string,
-    validationResults?: ValidationResult[]
-) {
-    let htmlReport = await ejs.renderFile('./template/report-template.ejs', {
+export async function generateReport(validationResults?: ValidationResult[]) {
+    const pathToTemplate = path.resolve(
+        __dirname,
+        path.join('..', '..', 'template', 'report-template.ejs')
+    );
```

Comment:
I think this could be simpliefied to `path.resolve(__dirname, '../../template/report-template.ejs')`

Reply:
That's right, path.resolve normalizes passed paths.